manim/mobject/svg/brace.py

```diff
@@ -164,7 +164,9 @@ class BraceText(BraceLabel):
 
 class BraceBetweeenPoints(Brace):
     """Similar to Brace, but instead of taking a mobject it uses 2
-    points to place the brace. A fitting direction for the brace is
+    points to place the brace. 
+    
```

Comment:
Ah well, I always forget about black when adding suggestions. Sorry, this fixes the black failure:

```suggestion
    points to place the brace.

```